contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,128 @@
+
+We're getting ready to make a substantial change to the way [persons](/manual/persons) and [events](/manual/events) work by essentially combining them and adding person ids and properties _onto_ events. 1.39.0 gives you the chance to try the feature ahead of the full release. Here's how:
+
+- **PostHog Cloud:** No action is needed, as we're rolling this test out steadily behind a feature flag. If you're eager to be included from the start, let us know in [Slack](/slack) to see about joining our beta group.
+-  **Self-hosted users:** First, you need to [run an async migration](/docs/runbook/async-migrations). Afterwards, change your `actors_on_events_teams` [instance setting](/docs/self-host/configure/instance-settings) to `true`. As with other instance settings, you'll need to be [a Staff user](/docs/self-host/configure/instance-settings#staff-users) to make changes.
+
+Why are we doing this? Feedback showed that users weren't able to create queries based on person properties at the time of an event. By putting person properties on events, this becomes the new default, while still enabling you to to filter insights based on the latest properties using cohorts. 
+
+Also, it makes queries significantly faster because we no longer have to merge tables to get a result; we can just look up everything in the events table instead. One query in our internal tests showed a 400x increase in speed...but this beta will help us understand the improvement in real-world conditions. 
+
+There'll be some slight changes to how filtering by unique users or person properties work as a result of this change. For more information on this and to find out how we've implemented it change behind the scenes, check out our [overview of the ingestion pipeline](/docs/architecture/ingestion-pipeline).
+
+### New: Format your insights to show durations
+
+![formatting an insight as a duration](../images/blog/array-1.39.0/duration-formatting.gif)
+
+Do your event properties contain milliseconds or seconds? Then you can now make things easier by changing your Y-axis unit to show durations instead of the default `none`. It's a small addition, but one which makes insights much easier to read and share with others.
+
+### New: Big, bold number chart type
+
+![the big bold number insight formatted as a duration](../images/blog/array-1.39.0/big-number.png)
+
+This new, much requested and _very_ exciting big bold number insight type does exactly what it says: simplifies a query result down to a single number. And it works great with durations too!
+
+### Improved: Turbo properties
+
+Frustrated that it was taking a few seconds to load properties? You weren't the only one. 
+
+Thankfully, version 1.39.0 addresses this by adding a limit so that PostHog doesn't always search through hundreds of thousands of properties for each query. The result? [Properties now load much faster] On PostHog Cloud we saw a 75% reduction in API call time. (https://github.com/PostHog/posthog/pull/11037). 
```

Comment:
```suggestion
Thankfully, version 1.39.0 addresses this by adding a limit so that PostHog doesn't always search through hundreds of thousands of properties for each query. The result? [Properties now load much faster](https://github.com/PostHog/posthog/pull/11037). On PostHog Cloud we saw a 75% reduction in API call time. 
```